src/cmd.rs

```diff
@@ -39,6 +39,11 @@ pub struct Dynein {
     #[structopt(short, long, global = true)]
     pub region: Option<String>,
 
+    /// Specifies the port number when with using `--region local`.
+    /// This option has an effect only when `--region local` is used.
```

Comment:
I think this is a little redundant description. I want to propose the following alternative.
```
Specify the port number. This option has an effect only when `--region local` is used.
```
What do you think?

Reply:
I think it's good 👍 
I'll change it.